### PR TITLE
Reenable ability to ignore bot user

### PR DESCRIPTION
**Pull request by mmitche**
Reenables the ability to ignore the github user that is used to make commens on PRs.  Also adds an option to enable this behavior, which is off by default (same behavior as today).

Also removed a it of logging that is very noisy in large installations.

### CODE REVIEW

**Comment by benpatterson** [comment]
@mmitche I'm signed on to help with this repo. I see you have multiple PRs. It will take me a little while to grok and get through everything. Is there a particular PR you suggest I dig into first? Cheers.

**Comment by mmitche** [comment]
This is probably the simplest one.  Thanks! 😃

**Comment by benpatterson** [comment]
ok cool i'll get started :)

**Comment by benpatterson** [comment]
I realize I don't have all the context here, but is there a legitimate reason that comments from the bot user _would_ be wanted by a job?

**Comment by mmitche** [comment]
There are two scenarios I could see:
1) You're running the PR plugin using your own credentials, and you want the PR builder to not ignore your comments
2) You're running as a bot account, but one that is shared among other infrastructure that might be posting to GH and attempting to control PR builder.

File: src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/help-ignoreCommentsFromBotUser.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div>
+	When checked, <em>GitHub Pull Request Builder</em> will ignore comments made
+    by the user associated with the PR builder credentials.  This means that comments
+    automatically made by the PR builder will cannot trigger new PR Builder actions.
```
Comment (benpatterson) [inline src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/help-ignoreCommentsFromBotUser.html]: typo: remove "will" from this line

**Comment by mmitche** [comment]
@benpatterson WIll do, thanks!

**Comment by mmitche** [comment]
@benpatterson Sorry it took so long. Updated with tests and fixed comment!

**Comment by mmitche** [comment]
@benpatterson Fixed the conflicts again, let me know if you need anything else.

**Comment by samrocketman** [comment]
@mmitche do you mind squashing this into a single commit and rebasing on latest master?

**Comment by mmitche** [comment]
@samrocketman Fixed

**Comment by bjoernhaeuser** [comment]
@mmitche sorry for the late response! 

Could you rebase again and see if this fixes the build error which is currently reported?

File: src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java [hunk 2]
```diff
@@ -136,8 +136,8 @@ public void perform(
         // ignore comments from bot user, this fixes an issue where the bot would auto-merge
         // a PR when the 'request for testing' phrase contains the PR merge trigger phrase and
         // the bot is a member of a whitelisted organization
-        if (helper.isBotUser(triggerSender)) {
-            listener.getLogger().println("Comment from bot user " + triggerSender.getLogin() + " ignored.");
+        if (helper.isIgnoreBotUser() && helper.isBotUser(triggerSender)) {
+			listener.getLogger().println("Comment from bot user " + triggerSender.getLogin() + " ignored.");
```
Comment (samrocketman) [inline src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java]: Mixing tabs and spaces.  Please stick to four space indents and not tabs.

**Pull request closed without merging**